preseed/provision.erb

```diff
@@ -139,7 +139,11 @@ tasksel tasksel/first multiselect minimal, ssh-server, openssh-server
 
 # Install some base packages
 d-i pkgsel/include string <%= puppet_package %> <%= salt_package %> lsb-release
+<% if /apt-/ =~ @host.name -%>
+d-i pkgsel/update-policy select none
+<% else -%>
 d-i pkgsel/update-policy select unattended-upgrades
```

Comment:
You should use a single line or statement here. See https://github.com/squarit/community-templates/blob/develop/preseed/provision.erb#L66 for an example. Please specify a more generic name for the variable as well. Ex: `@host.params['preseed-update-policy']`

The value can then be directly set for ubuntu host groups in foreman.